### ISSUE TITLE
add top-two instant runoff voting tabulation

### ISSUE THREAD

**Issue by chughes297** [BrightSpots/rcv#634]
add a winning method that produces a top-two IRV tabulation, where: 
first choices are tallied
top two candidates are identified
all candidates except the top two are eliminated
Ballots transfer from eliminated candidates to the top two candidates
candidate with the most votes wins

**Comment by endolith** [comment]
This is called "Contingent Vote" and has even more problems than Hare RCV: https://en.wikipedia.org/wiki/Contingent_vote

See https://github.com/BrightSpots/rcv/issues/579

**Comment by tarheel** [comment]
@chughes297: are there specific jurisdictions that need this? (Assuming yes since you listed it as a v1.4 requirement.)

@endolith: yeah, you're preaching to the choir here! But I think our general stance is that a flawed version of IRV is better than nothing (FPTP) and we're willing to support it in the app if that's what it takes to get a jurisdiction moving in the right direction. (But that's just my take. @chughes297 can speak on behalf of the RCV Resource Center.)

**Comment by chughes297** [comment]
Agree, I am NOT a fan but [Utah has a bill](https://le.utah.gov/~2023/bills/hbillint/HB0205S01.pdf) that apparently has legs and requires this. 

We could leave them high and dry....

**Comment by andyanderson** [comment]
Utah — Are they not already high and dry? 😂

**Comment by chughes297** [comment]
I heard the Utah bill has died so I am removing the 1.4 req label.
